Assignment1/assignment2/lib/utilities.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:assignment2/users.dart';
+import 'dart:collection';
+
+Map<int, User> currentSession = Map();
```

Comment:
This can be separated out into a different file/class named something like  SessionStorage. Should not be part of utilities.

Reply:
The new class can contain methods to manipulate the map

Reply:
done

Reply:
done